DistroLauncher/WSLInfo.h

```diff
@@ -72,7 +72,7 @@ namespace Oobe::internal
             }
             const std::wstring filename{entry.path().filename().wstring()};
             return std::any_of(std::begin(elements), std::end(elements), [&filename](const auto& element) {
-                return starts_with({filename}, element) && ends_with({filename}, L".snap"sv);
+                return starts_with(filename, element) && ends_with(filename, L".snap"sv);
```

Comment:
I think the `L".snap"sv` can drop the literal suffix, can't it?

Reply:
Yes indeed. Also the `using namespace std::literals::string_view_literals
` directive. Changed.